fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java

```diff
@@ -264,6 +264,12 @@ public void updateRoles(final Set<Role> allRoles) {
 
         final String usernameParamName = "username";
         if (command.isChangeInStringParameterNamed(usernameParamName, this.username)) {
+
+            // TODO Remove this check once we are identifying system user based on user ID
```

Comment:
Do we have an issue explaining how to implement this todo or just to log the issue on Jira so someone can pick it up later?

Reply:
I've created a new issue for this - FINERACT-1068